orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -225,6 +227,13 @@ class PackageInfo {
     }
   }
 
+  @CompileDynamic
+  private Map filterNUPKGArtifacts(List<Map> artifacts, String prefix) {
```

Comment:
Looks like this method does the same thing as the `else` branch at the call-site. Is it necessary to add this method?